csv_generator/consumers/utils.py

```diff
@@ -0,0 +1,59 @@
+import pendulum
```

Comment:
Didn't know about `pendulum`. Just wondering how it compares to `dateutil`?

Reply:
I have gone between `pendulum`, `arrow` and `dateutil` in the past, just prefer its interface, docs and some of the sugar it gives you. Though here I am only really using it for its extra string formatting tokens.